### PR TITLE
Create genre class

### LINKED ISSUE #7: [1pt] Create Genre class
- [x] Create Genre class with an association to the Item class (in a separate .rb file).
- [x] All Genre class properties visible in the diagram should be defined and set up in the constructor method.

### PR DESCRIPTION
**Pull request by Bavon101**
Hello, as per this PR I achieved the following:

**Project requirements**
- [x] Create Genre class with an association to the Item class (in a separate .rb file).
- [x] All Genre class properties visible in the diagram should be defined and set up in the constructor method.

### CODE REVIEW

File: lib/genre.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_accessor :name
+
+  def initialize(name)
+    @id = rand(1..10_000)
+    @name = name
+    @items = []
+  end
```
Comment (ahangarha) [inline lib/genre.rb]: I think 'items' should have a getter.

**Review by ahangarha — CHANGES_REQUESTED** [review]
Very close...

## CHANGE REQUIRED :warning: 



Everything looks good.
- No linter error
- Clean code

There is one issue which is needed to be fixed as per project requirement. Please check the comment. It is tiny issue but important. If there is any question, please ask under the comment.

File: lib/genre.rb (hunk 1 above)
Comment (Bavon101) [reply lib/genre.rb]: @ahangarha Thanks for the catch, I've updated the code and pushed it, please do check.

**Review by ahangarha — APPROVED** [review]
Now...

## APPROVED :tada: 



Thanks for fixing the issue.  
Now the PR fulfills all the requirements. So feel free to merge and continue with the next task.

Well done :+1:

File: lib/genre.rb (hunk 1 above)
Comment (ahangarha) [reply lib/genre.rb]: I myself noticed this issue because of TDD :laughing:

**Pull request merged**